app/scripts/metamask-controller.js

```diff
@@ -1954,16 +1954,14 @@ export default class MetamaskController extends EventEmitter {
    * Create a new Vault and restore an existent keyring.
    *
    * @param {string} password
-   * @param {number[]} encodedSeedPhrase - The seed phrase, encoded as an array
+   * @param {string} seedPhrase - The seed phrase, encoded as an array
    * of UTF-8 bytes.
```

Comment:
nit: we can update the comment description now that we've updated the type passed in